### DISCUSSION TITLE
Location and Site Difference (Toyota Technological Institutes)

### DISCUSSION THREAD

**Discussion by Yasue33** [community/community#51474] [GitHub Education]
I am a "Toyota Technological Institute" student. (toyota-ti.ac.jp, TTI)
I tried to sign up Github Education but I was rejected by the location difference.
I checked my university in the list and university's URL is different.
Now it was our sister university "Toyota Technological Institute at Chicago". (ttic.edu, TTIC)
Please correct TTI's URL and location information.

**Comment by samus-aran** [comment]
Hi @Yasue33 👋 

Thank you for bringing this to our attention. We'll pass this onto our Education team and if we have any updates we'll post here.

**Comment by Yasue33** [comment]
Thank you for updating the locational information!
I succeeded to register.

**Reply by samus-aran** [reply]
Sorry @Yasue33 didn't get a chance to update this thread but I'm glad you were able to successfully register 🎉 Enjoy your time on GitHub Global Campus 🥇
